tests/devices/unit_tests/conftest.py

```diff
@@ -31,3 +36,21 @@ def patch_motor(motor: Motor, initial_position=0):
 @pytest.fixture
 def static_directory_provider(tmp_path: Path) -> DirectoryProvider:
     return StaticDirectoryProvider(tmp_path)
+
+
+@pytest.fixture
+def smargon():
```

Comment:
Could: There is a better sim motor now in `ophyd_async.sim.demo.sim_motor` which we could replace the `Motor`s with instead